util/test/tests/Vulkan/Vk_Pixel_History.py

```diff
@@ -0,0 +1,130 @@
+import renderdoc as rd
+import rdtest
+
+def value_selector(x): return x.floatValue
+def passed(x): return x.Passed()
+def event_id(x): return x.eventId
+def culled(x): return x.backfaceCulled
+def depth_test_failed(x): return x.depthTestFailed
+def stencil_test_failed(x): return x.stencilTestFailed
+def shader_discarded(x): return x.shaderDiscarded
+def shader_out_col(x): return value_selector(x.shaderOut.col)
+def post_mod_col(x): return value_selector(x.postMod.col)
+def primitive_id(x): return x.primitiveID
+
+class VK_Pixel_History(rdtest.TestCase):
+    demos_test_name = 'VK_Pixel_History_Test'
+    demos_frame_cap = 5
+
+    # Need to enable Vulkan Pixel History before opening the capture
+    def run(self):
+        obj: rd.SDObject = rd.SetConfigSetting("Vulkan.PixelHistory")
```

Comment:
We shouldn't force on settings like this in tests. It doesn't currently get saved automatically without an explicit save but that may change in the future, and it also means this test will be running for everyone even on a beta feature.

Instead since the settings are now available anywhere (not just in the UI) we should let the user enable it if they wish and have the existing check below where we don't run the test if the APIProperties show it as unsupported.

Reply:
Done. I copied from VK_RGP_Capture before.
Do you run these tests automatically? If so, should I turn on the setting for pixel history so it gets tested at some point?

Reply:
Ah, yes that goes back to when these settings weren't available outside the UI, and RGP integration is not a beta feature but it's disabled by default due to stability concerns. I should be able to remove that and allow it to work the same now that the settings are available everywhere.

Yes the tests are run as part of the nightly build process: https://renderdoc.org/test_results

For beta features like this the testing won't be enabled until they ship by default, so that's not an issue.